experimental/dds/tree2/src/class-tree/tree.ts

```diff
@@ -58,7 +58,7 @@ export interface ITree extends IChannel {
 }
 
 /**
- * @alpha
+ * @beta
```

Comment:
Can we document this while we're here?

Reply:
I do not think changes which we actually want to merge, like improved docs should be included in this PR since that means those changes likely will never get merged (see https://github.com/microsoft/FluidFramework/pull/17865 for an example of that happening).

While I think it would be less work to merge this, then refine the API, choices about how we do that are up to the API review council, and I don't want docs improvements to be gated on their slower review path.